spec/recommendation/engine_spec.rb

```diff
@@ -1,35 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe 'Testing' do
-  describe 'say_hello' do
-    xit 'can run python' do
-      cocktail_1 = create(:cocktail, name: 'margarita')
-      cocktail_2 = create(:cocktail, name: 'old fashion')
-      cocktail_3 = create(:cocktail, name: 'whisky sour')
-
-      create(:user) do
-        create(:rating, cocktail: cocktail_1, stars: 2)
-        create(:rating, cocktail: cocktail_2, stars: 2)
-        create(:rating, cocktail: cocktail_3, stars: 2)
-      end
-
-      create(:user) do
```

Comment:
Let's be sure you're merging in main to your branch before merging this branch to main @Jaybraum ! We don't want to lose these files, but we need to resolve some conflicts. Happy to help with that process, if needed, since I should have a good idea what to keep or not.